freelist.go

```diff
@@ -49,7 +96,12 @@ func (f *freelist) count() int {
 
 // free_count returns count of free pages
 func (f *freelist) free_count() int {
-	return len(f.ids)
+	//use the forwardmap to get the total count
```

Comment:
how often will this func be called? probably consider add a counter to record the number of free pages.